### PR TITLE
Added serialization for player locations.

### PR DESCRIPTION
**Pull request by patmay**
This should pass tests after #91 has been merged into integ and this ticket rebased.

Update: this has been rebased and should be good to go.

### CODE REVIEW

**Review by stephensmeal — DISMISSED** [review]
Sure I'll approve it, why not?

File: test/unit/test_playerlist.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from src.card import Card
 from src.cardtype import CardType
 from src.playerstatus import PlayerStatus
-
+from src.gamestate import GameState
```
Comment (stephensmeal) [inline test/unit/test_playerlist.py]: Should this be removed? Isn't it residual from the original commit?

File: src/playerlist.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 """playerlist.py"""
 from src.player import Player
 from src.playerstatus import PlayerStatus
+import json
```
Comment (stephensmeal) [inline src/playerlist.py]: Should this be removed? Isn't it residual from the original commit?

**Review by stephensmeal — COMMENTED** [review]
Good enough I guess.

**Review by stephensmeal — APPROVED** [review]
Good enough I guess.

**Comment by yo09975** [comment]
Ticket has been merged. Must be a bug since there were originally multiple 80_ branches.

**Pull request closed without merging**